utils.js

```diff
@@ -11,15 +11,14 @@
  * for the specific language governing rights and limitations under the
  * License.
  *
- * The Original Code is Home Dash Utility.
+ * The Original Code is Vertical Tabs.
  *
- * The Initial Developer of the Original Code is The Mozilla Foundation.
+ * The Initial Developer of the Original Code is
+ * Philipp von Weitershausen.
  * Portions created by the Initial Developer are Copyright (C) 2011
  * the Initial Developer. All Rights Reserved.
  *
  * Contributor(s):
- *   Edward Lee <edilee@mozilla.com>
- *   Erik Vold <erikvvold@gmail.com>
```

Comment:
Changing the attribution and contributors of this code seems like an odd thing to do…